src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -603,7 +603,8 @@ private void InitializeTrainingGraph(IDataView input)
                     (string)labelType.ToString());
             }
 
-            Contracts.CheckParam(labelCount > 1, nameof(labelCount), "ImageClassificationTrainer requires more than 1 class in the training dataset");
+            var msg = $"Only one class found in {_options.LabelColumnName} column. To build a multiclass classification model, the number of classes needs to be 2 or greater";
```

Comment:
Slight reword to my original: var msg = $"Only one class found in *the* {_options.LabelColumnName} column.